tests/examples/test_recipient_features_stage.py

```diff
@@ -0,0 +1,63 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Move this file into a subdirectory under `tests/examples` to mirror the example it is testing (`developer_guide`)